recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     - patches/0001-Fix-software-version-test.patch
 
 build:
-  number: 2
+  number: 3
   script: |
     {{ PYTHON }} -m pip install . -vv
   skip: True  # [win or (aarch64 and py<39)]
```

Comment:
This appears to disable Python pre-3.9 on ARM. Looks like that was added in PR ( https://github.com/conda-forge/pyorc-feedstock/pull/16 )

@noirello, do you recall why this was needed?

Reply:
It had to be that. I wanted to double check, but the logs are no longer available.

Reply:
Yeah unfortunately the logs get removed after 30 days